### ISSUE TITLE
Lens model is not displayed when read in from json

### ISSUE THREAD

**Issue by drphilmarshall** [linan7788626/LensWranglerPlusPlus#3]
I see the critical curves and caustics, but not the adjustable model elements - these are needed so that user 2 can improve on user 1's model

**Comment by linan7788626** [comment]
Hi @drphilmarshall, 

This is a key point of the APP, I tried to show the critical curves and caustics when I loaded in a JSON file, but I did not make it. If you could please to give me any suggestions, I appreciate it.

Thanks,
Nan

**Comment by drphilmarshall** [comment]
Well, one thought is that this functionality is connected to that mentioned in #5 (current model should be displayed in all panels), so addressing #5 first could be the way to go?

**Comment by linan7788626** [comment]
Yes, you are right. I made a mistake in the reply; I wanted to say I could not see the mass model and source model after I loaded the JSON file.  And I agree on the connection between this issue and #5, so I will focus on #5  as you suggested. Thanks.

**Comment by linan7788626** [comment]
Hi @drphilmarshall ,

I tried to display ellipses in Mass model block and source model block when I loaded the JSON model, but I did not make it, it is more complicated than I thought. If you have any suggestions, please feel free to let me know, I really appreciate it. I would like to focus on this issue in coming weeks.

Thanks,
Nan

**Comment by drphilmarshall** [comment]
Hmm. Without looking at the code, I'd say the following: this operation
_should_ be easy, because the action involved is _conceptually
straightforward_. The fact that coding it is more complicated than we
thought it would be suggests that we need to refactor the code until it
_is_ easy. That probably means redesigning the objects and how they
interact with each other.

**Comment by drphilmarshall** [comment]
Hey @kapadia! Long time no see :-) Nan Li, a very good postdoc at Argonne National Lab, has gotten interested in our LensWrangler code, and extended it into a more advanced modeler called "Hoopla" (geddit? :-) ). We'll be fiddling with adjustable mass models, and I thought you'd be interested to follow along :-)
